### PR TITLE
To Do List: Add module branch.

### PR DESCRIPTION
**Pull request by Abmart0803**
Added module branch to implement clearAll function in .src folder.

### CODE REVIEW

**Review by Reem-lab — APPROVED** [review]
# Hi @Abmart0803,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥💯💯


## Highlights 🔥🔥🔥🔥
- Followed GitHub flow ✅
- No Linters error 🤩🔥💯
- Great working on the functionality🔥🔥
- Descriptive PR 💯💯
- Professional README 🔥🔥



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          <li class="uil-refreshe refreshe">
+            <button id="add">
+              <iconify-icon icon="bx:refresh"></iconify-icon>
+            </button>
+          </li>
+        </div>
+        <ul id="task-list"></ul>
+        <div id="clear">
+          <button id="clearAll">Clear All completed</button>
+        </div>
```
[the remark this answers is not in the document]
Comment (luigirazum) [reply dist/index.html:32]: Hello @Reem-lab, I think you tagged the wrong person.
Shouldn't you tag @Abmart0803 ?

Fortunately the project was approved. 👍 
Happy Coding @Abmart0803 !! 👨‍💻 
Happy Reviewing @Reem-lab !! 🔍

[the remark this answers is not in the document]
Comment (Reem-lab) [reply dist/index.html:32]: Hello @luigirazum  it was by mistake sorry 😊🙏🏻

[the remark this answers is not in the document]
Comment (luigirazum) [reply dist/index.html:32]: Don't worry, but I got scared a little...

See you and Happy Reviewings

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+      },
+      {
+        description: 'Service Car',
+        completed: true,
+        index: 1,
+      },
+      {
+        description: 'Finish To do List',
+        completed: true,
+        index: 2,
```
Comment (Abmart0803) [inline src/index.js:171]: This lines of codes can be refactor.

**Pull request merged**